coordinape/coordinape
# Hasura Delete epoch endpoint

### Test Plan
1. Go to http://localhost:3000/admin/circles and create an epoch in the future.
2. Run this mutation in your hasura console
```gql
mutation DeleteMutation {
  deleteEpoch(payload: {circle_id: <circle_id>, id: <epoch_id>}) {
    success
  }
}
```
3. refresh the page http://localhost:3000/admin/circles , the epoch that you created should be gone now.
